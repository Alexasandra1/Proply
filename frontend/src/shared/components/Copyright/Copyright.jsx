import React, { useState } from "react"
// import logotip from "../../../image/logoLitlib.png"

import './Copyright.scss'

export function Copyright(props) {

    return (

        <div className="copyright">
            <div className={`copyright__text ${props.copyrightStyle}`}>{props.copyrightWord}</div>
        </div>

    )
}