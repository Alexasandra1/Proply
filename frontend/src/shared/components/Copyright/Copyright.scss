.copyrightStyle {
    color: #FFF;
    font-family: Montserrat;
    font-size: 20px;
    font-style: normal;
    font-weight: 600;
    justify-content: center;
    text-align: center;
}