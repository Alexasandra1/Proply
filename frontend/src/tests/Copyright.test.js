import React from 'react';
import renderer from 'react-test-renderer';
import { Copyright } from '../shared/components/Copyright/Copyright';

describe('Copyright', () => {
    it('renders correctly', () => {
        const tree = renderer
            .create(<Copyright copyrightStyle="style" copyrightWord="word" />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});
