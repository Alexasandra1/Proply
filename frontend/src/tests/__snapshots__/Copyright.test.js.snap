// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Copyright renders correctly 1`] = `
<div
  className="copyright"
>
  <div
    className="copyright__text style"
  >
    word
  </div>
</div>
`;
